sunpy/timeseries/sources/goes.py

```diff
@@ -89,6 +89,8 @@ def plot(self, axes=None, columns=None, **kwargs):
         axes, columns = self._setup_axes_columns(axes, columns)
         plot_settings = {"xrsa": ["blue", r"0.5--4.0 $\AA$"], "xrsb": ["red", r"1.0--8.0 $\AA$"]}
         data = self.to_dataframe()
+        # We do not want to plot the quality flags in the light curve.
+        columns = [col for col in columns if col not in ["xrsa_quality", "xrsb_quality"]]
         for channel in columns:
```

Comment:
Sorry, my bad for merging that PR without checking... Might be a bit nicer to do this instead?
```suggestion
        for channel in ['xrsa', 'xrsb']:
```

Reply:
That is so much simpler, I don't know why I didn't do that.